CMakeLists.txt

```diff
@@ -23,31 +23,51 @@ set(HEAPTRACK_FILE_FORMAT_VERSION 2)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
 include(FeatureSummary)
-find_package(Libunwind REQUIRED)
-find_package(Boost 1.41.0 COMPONENTS system filesystem) # option first
-find_package(Boost 1.41.0 REQUIRED COMPONENTS iostreams program_options) # then required
+find_package(Boost 1.41.0 COMPONENTS system filesystem iostreams)
 find_package(Threads REQUIRED)
 find_package(ZLIB REQUIRED)
-find_package(Zstd)
+
+if (${BOOST_iostreams_FOUND})
+    find_package(Zstd)
+endif()
 set_package_properties(Zstd PROPERTIES TYPE RECOMMENDED PURPOSE "Zstandard offers better (de)compression performance compared with gzip/zlib, making heaptrack faster and datafiles smaller.")
 
+if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
+  set(HEAPTRACK_BUILD_TRACK_DEFAULT ON)
+else()
+  set(HEAPTRACK_BUILD_TRACK_DEFAULT OFF)
+endif()
+
 option(
-  HEAPTRACK_BUILD_GUI
-  "Disable this option to skip building the Qt5 / KF5 based GUI for heaptrack."
+  HEAPTRACK_BUILD_TRACK
+  "Disable this option to skip building the tracker part for heaptrack, e.g. to only build the GUI."
+  ${HEAPTRACK_BUILD_TRACK_DEFAULT}
+)
+
+option(
+  HEAPTRACK_BUILD_TESTS
```

Comment:
shouldn't be needed, afaik there's BUILD_TESTING which is defaulted to on. You can specify that to off if needed already, no?

Reply:
That's right. I changed it